src/clib/pio_nc.c

```diff
@@ -394,9 +394,13 @@ int PIOc_inq_type(int ncid, nc_type xtype, char *name, PIO_Offset *sizep)
         LOG((2, "PIOc_inq_type netcdf call returned %d", ierr));
     }
 
-    /* Failure to inquire is fatal */
-    ierr = check_netcdf(NULL, file, ierr, __FILE__, __LINE__);
-    if(ierr != PIO_NOERR){
+    /* A failure to inquire is not fatal */
```

Comment:
I had determined before that certain inq function failures were fatal. For example, inquiring the name/size of a type (PIOc_inq_type() ) .
So I think in this PR we only need to modify the cases where the comments specifically mention "Failure to inquire is not fatal"
There are several cases below that need to be similarly reverted back.